### PR TITLE
feat(BAC-60): add fmc push notification func

### PR DESCRIPTION
**Pull request by Chiefkufre**
TICKET: https://linear.app/team-gear/issue/BAC-60/push-weather-alerts

TASK DESC:
 Create a function to push weather alerts firebase cloud messaging topics.

-  *send_notification_to_topic(payload, topic)->** function receives payload when available and send alerts to all platform clients-web, mobile, etc

Status: Done..available for testing

### CODE REVIEW

**Comment by Chiefkufre** [comment]
add some changes

**Comment by devvspaces** [comment]
Hi, your time is to return `YYYY-MM-DD H:M`, but you used `(payload['start'], '%d/%m/%y %H:%M:%S')`

**Comment by devvspaces** [comment]
also use `payload['end']` instead of `payload['start']`

**Comment by devvspaces** [comment]
You can use this @Chiefkufre 

image

File: backend/app/utils/push_notification.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+firebase_admin.initialize_app(cred)
+
+def send_notification_to_topic(payload, topic):
+  
+    # See documentation on defining a message payload.
+    message = messaging.Message(
+        data={
+        "event": payload['event'],
+        "message": payload["description"],
+        "datetime": datetime.strptime(payload['end'], '%y-%m-%d %H:%M')
```
Comment (devvspaces) [inline backend/app/utils/push_notification.py]: not small letter y, it's capital letter Y sir

**Review by devvspaces — COMMENTED** [review]
Okey dokey

**Pull request merged**